#!/usr/bin/env python
PACKAGE = "lidar_obstacle_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("safety_region_xmin",     double_t, 0, "Default: -2.0",  -2.0,  -5.0,  0.0)
gen.add("safety_region_xmax",     double_t, 0, "Default: 2.0",    2.0,   0.0,  5.0)

gen.add("safety_region_ymin",     double_t, 0, "Default: 0.2",    0.2,   0.0,  5.0)
gen.add("safety_region_ymax",     double_t, 0, "Default: 7.0",    7.0,   0.0,  10.0)

gen.add("obstalce_size_thres",    double_t, 0, "Default: 0.1",    0.1,   0.0,  1.0)

exit(gen.generate(PACKAGE, "safety_checker", "safety_checker_"))